### DISCUSSION TITLE
DDS SBN Plugin

### DISCUSSION THREAD

**Discussion by swhart115** [nasa/cFS#506] [Q&A]
Hi,
I've heard rumors that there has been an effort to create a DDS plugin for SBN but, other than a reference to it from some 2017 presentations from the CHREC project, i have not been able to find any further information. Is this something anyone knows anything about and can point me in the right direction on?
Thanks,
Steve

**Comment by ivanperez-keera** [comment]
Is this related to the work on a ROS - cFS bridge by JAXA, or the one by TRACLabs?

**Reply by ana-GT** [reply]
I think Steve is referring to a non-TRACLabs bridge (and I think you are correct, Ivan, he is probably referring to the JAXA bridge).
